src/components/dialogs/DialogGatewayValue.vue

```diff
@@ -172,7 +176,7 @@
           color="blue darken-1"
           text
           @click="$refs.form.validate() && $emit('save')"
-          >Save</v-btn
+          >Update</v-btn
```

Comment:
We could make this to be `Add` or `Update` based on if it's a new or an existing value

Reply:
The only way I've seen to detect a new or existing value is to check, if editedValue is an empty object when the dialog is shown and store that in a data property (later it's not empty anymore).
Or do you know a better way to detect this (without adding a new component property)?

Reply:
done.